components/Collections/src/MDCCollectionViewController.h

```diff
@@ -49,9 +49,6 @@
  as well as ink during the highlight/unhighlight states.
  */
 
-- (BOOL)collectionView:(nonnull UICollectionView *)collectionView
```

Comment:
The [default implementation is to return YES](https://developer.apple.com/documentation/uikit/uicollectionviewdelegate/1618070-collectionview?language=objc). Do you see an advantage in us explicitly returning YES as well?